build/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM registry.access.redhat.com/ubi7-dev-preview/ubi-minimal:7.6
+FROM alpine:3.10
```

Comment:
Let's see if base image for the operator can use [distroless](https://github.com/GoogleContainerTools/distroless) instead of alpine.

Reply:
I'm pretty sure it can, but I wanted to keep the same base for the runners and the operator itself. This way we only have to maintain one. Thoughts?

Reply:
If we are going to switch to a slimmed down based container image then I would prefer to use the distroless base image for the operator image.

I'm fine with having the job pods use a separate base images as those currently require a shell.

Additionally keep in mind that if we switch to a non-RHEL like base image then we cannot get certified. I'm not concerned about this, but just letting you know.

https://github.com/operator-framework/operatorhub-docs/blob/master/distributing-an-operator/distributing-an-operator/certification-requirements.md

Reply:
Valid point. Not sure how much we care about the certification right now. We can always switch back if we have to.
Can we make the distroless a separate issue? It needs a little bit of testing and I don't think I'll get to that in the next couple of days.

Reply:
I think we should just do it right the first time. No need to switch to alpine and then switch to distroless later. Let's take some and test properly instead of rushing through it.